src/app/customer-search/customer-search.component.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, OnInit, Injector, ComponentFactoryResolver, ApplicationRef } from '@angular/core';
+import { TestsDataService } from '../services/tests-data.service';
+import { TsScreenerDataService } from '../services/ts-screener-data.service';
+import { ThsDataService } from '../services/ths-data.service';
+import { TfiDataService } from '../services/tfi-data.service';
+
+@Component({
+  selector: 'customer-search',
+  templateUrl: './customer-search.component.html',
+  styleUrls: ['./customer-search.component.css']
+})
+export class CustomerSearchComponent implements OnInit {
+  public idSearch: string;
+  public searchBtn: boolean = true; // Search button is disabled while querying DB
+  public resultsTable = [];
+  public currentPage: number = 0;
+
+  constructor(
+    private testDataService: TestsDataService, private tsDataService: TsScreenerDataService, private thsDataService: ThsDataService, private tfiDataService: TfiDataService)
+    { }
+
+  ngOnInit() {
+  }
+
+  // Gets JSON from queryDB() puts it into an array to create the table of appointments
+  // sorts by date, most recent first.
+  public patientSearch() {
```

Comment:
There should be a valid Search option here (I can search for invalid patients if I click the search button).